ui/pages/add-nft/add-nft.js

```diff
@@ -100,19 +101,19 @@ export default function AddNft() {
     const tokenDetails = await getTokenStandardAndDetails(
       nftAddress,
       null,
-      tokenId.toString(),
+      tokenId.toString(), // this should already be a string?
     );
 
     trackEvent({
-      event: MetaMetricsEventName.TokenAdded,
-      category: 'Wallet',
+      event: MetaMetricsEventName.NftAdded,
+      category: MetaMetricsEventCategory.Wallet,
       sensitiveProperties: {
         token_contract_address: nftAddress,
         token_symbol: tokenDetails?.symbol,
-        tokenId: tokenId.toString(),
-        asset_type: AssetType.NFT,
+        token_id: tokenId.toString(), // This should already be a string?
```

Comment:
does this need to be `.toString()`ed?

Reply:
Looks like you're right, but could you verify by testing it manually just to be absolutely sure before we remove those `toString()`s?

Reply:
verified manually. not needed